docs/tutorial/6-charm-level-tracing.md

```diff
@@ -0,0 +1,113 @@
+# Set up Charm-Level Tracing
+
+While developing and debugging, it can be helpful to see traces of the MAAS Site Manager charm's workload. In this section, we will set up some necessary Juju components and relate them to MAAS Site Manager K8s in order to see these traces.
+
+[note]
+**Note**: You must have completed the last section if you would like to enable charm-level tracing.
+[/note]
+
+Before beginning, ensure you are working in the `cos-lite` model by running:
+
+```bash
+juju switch cos-lite
+```
+
+## Deploy Tempo K8s
+
+First, deploy the Tempo Coordinator and Worker charms, and wait for them to report as `blocked/idle`:
+
+```bash
+juju deploy tempo-coordinator-k8s --channel edge --trust tempo
+juju deploy tempo-worker-k8s --channel edge --trust tempo-worker
+juju status --watch 5s
+```
+
+Next, integrate the coordinator and worker charms:
+
+```bash
+juju integrate tempo tempo-worker
+```
+
+## Deploy Minio and S3
```

Comment:
why Minio? we have a S3 service already from step 3

Reply:
This is just copied from the charm-level tracing tutorial. I suppose we could re-use the s3-integrator we already deployed, however, I haven't tested it (and this uses the `edge` channel while we use the `latest/stable`, but maybe it still works). Or, we could just link to the tutorial

Reply:
if you have external docs that you can refer to, do it. let's avoid duplicating information